src/main/webapp/WEB-INF/views/cars/add.jsp

```diff
@@ -0,0 +1,16 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add car</title>
+</head>
+<body>
+<h1>Create car</h1>
+<form method="post" action="${pageContext.request.contextPath}/cars/add">
+    Model <input type="text" name="model"><br>
+    Manufacturer ID <input type="text" name="manufacturer_id"><br>
```

Comment:
is there a better input type for manufacturer id?
https://www.w3schools.com/html/html_form_input_types.asp